### PR TITLE
Update driver load stress cfg file

### PR DESCRIPTION
**Pull request by suqinhuang**
1. Update the cfg base on the update of balloon_stress cfg file
    2. Set the balloon_test flag in the get_memory_boundary,
    as the balloon operation is executed, and becasue the boundary balloon time
    is different for different mem, which will make the wait_time different.
id: 1426039

### CODE REVIEW

File: qemu/tests/balloon_check.py [hunk 1]
```diff
@@ -181,7 +181,9 @@ def get_memory_boundary(self, balloon_type=''):
         min_size = int(float(utils_misc.normalize_data_size(min_size)))
         if self.params.get('os_type') == 'windows':
             logging.info("Get windows miminum balloon value:")
+            self.env["balloon_test"] = 0
             self.vm.balloon(1)
+            self.env["balloon_test"] = 1
```
Comment (huangyum) [inline qemu/tests/balloon_check.py]: What's the point to set the flag ?  I didn't see other codes using this flag.

**Comment by suqinhuang** [comment]
@kimi1978wy @peixiu could you help review this patch

**Comment by kimi1978wy** [comment]
(1/3) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.driver_load_stress.with_balloon.before_bg_test: PASS (633.07 s)
 (2/3) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.driver_load_stress.with_balloon.during_bg_test: PASS (512.49 s)
 (3/3) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.driver_load_stress.with_balloon.after_bg_test: PASS (499.27 s)

Ack by : Yu Wang <wyu@redhat.com>

**Comment by peixiu** [comment]
cli: python ConfigTest.py --testcase=driver_load_stress.with_balloon --guestname=Win2016 --platform=x86_64 --clone=no

Results:
 (1/3) smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.driver_load_stress.with_balloon.before_bg_test: PASS (561.65 s)
 (2/3) smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.driver_load_stress.with_balloon.during_bg_test: PASS (383.44 s)
 (3/3) smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.driver_load_stress.with_balloon.after_bg_test: PASS (372.16 s)

Acked-by: Peixiu Hou phou@redhat.com

**Comment by suqinhuang** [comment]
Thanks @peixiu @kimi1978wy merge this patch

**Pull request merged**